src/index.html

```diff
@@ -11,7 +11,191 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form 
+       action="https://mate-academy-form-lesson.herokuapp.com/ create-application"    
+       method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label for="surname" class="form__field">
+            Surname:
+          <input 
+             type="text" 
+             autocomplete="off"
+             class="surname" 
+             name="surname" 
+             required
+             id="surname"
+          >
+        </label>
+
+        <label for="name" class="form__field">
+            Name:
+          <input 
+             type="text" 
+             autocomplete="off"
+             class="name" 
+             name="name" 
+             required
+             id="name"
+          >
+        </label>
+
+        <label for="howoldyou" class="form__field">
+            How old are You?
+          <input 
+             type="number" 
+             class="field" 
+             name="howoldyou" 
+             min="12"
+             max="100"
+             id="howoldyou"
+             value="12"
+          >
+        </label>
+
+        <label for="dateofbirth" class="form__field">
+            Full date of birth:
+          <input 
+             type="date" 
+             autocomplete="off"
+             class="field" 
+             name="dateofbirth" 
```

Comment:
```suggestion
             name="dateOfBirth" 
```
Don't forget to use camelCase, so names are easier to read
![image](https://user-images.githubusercontent.com/96409563/222356422-1cd1b54a-38c7-40e1-89d8-936025f316e4.png)